content/kubeone/master/guides/local/_index.en.md

```diff
@@ -0,0 +1,62 @@
++++
+title = "Local subcommand"
```

Comment:
Let's find a better title for this document. What about something like Creating a local all-in-one cluster?

Reply:
That title will be shown is sidebar menu, and such a long like would word-wrap around and make it ugly AF.

Reply:
What about `Local installation` instead?

Reply:
Without context it looks like it's about "installing kubeone binary locally", a bit misleading.

Few options from the top of my head:
* `One Node Cluster`
* `All-in-one Cluster`
* `Edge Node`
* `Edge Cluster`

Reply:
Maybe `All-in-one Cluster` is a good solution because we already use that wording in the document.